test/integration/features/basic.feature

```diff
@@ -102,15 +102,6 @@ Feature: Basic
       hello
       """
 
```

Comment:
@praveenkumar which seems to be moot. I could specify anything on the command line, but when untested they are not for sure relevant or correctly implemented. If you remove the previous test, I see no relevance to keep this in now. If however we will use this, we need to be sure it works... also for the CentOS image.